recipes/pyramid/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "pyramid" %}
+{% set version = "1.10.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 37c3e1c9eae72817e0365e2a38143543aee8b75240701fa5cb3a1be86c01a1c0
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pcreate=pyramid.scripts.pcreate:main
+    - pserve=pyramid.scripts.pserve:main
+    - pshell=pyramid.scripts.pshell:main
+    - proutes=pyramid.scripts.proutes:main
+    - pviews=pyramid.scripts.pviews:main
+    - ptweens=pyramid.scripts.ptweens:main
+    - prequest=pyramid.scripts.prequest:main
+    - pdistreport=pyramid.scripts.pdistreport:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
```

Comment:
You probably only need `pip`, `python` and `setuptools` here.

Reply:
Yeap. Leave only required deps. Removed setuptools from run also.